controller/controller.go

```diff
@@ -222,27 +222,20 @@ func (c *Controller) add(obj Object) {
 	defer c.Unlock()
 
 	c.cache.Add(obj)
-	c.propagate([]ResourceEvent{{obj.GetObjectKind().GroupVersionKind().GroupKind(), CreateEvent, nil, obj}})
 }
 
-func (c *Controller) update(oldObj, newObj Object) {
+func (c *Controller) update(_, newObj Object) {
```

Comment:
This makes _update_ and _create_ virtually identical, which is not ideal. See #24.